habitat/config/default.py

```diff
@@ -218,6 +248,12 @@ def __init__(self, *args, **kwargs):
 _C.SIMULATOR.TURN_ANGLE = 10  # angle to rotate left or right in degrees
 _C.SIMULATOR.TILT_ANGLE = 15  # angle to tilt the camera up or down in degrees
 _C.SIMULATOR.DEFAULT_AGENT_ID = 0
+_C.SIMULATOR.INITIAL_LOOK_DOWN_ANGLE = (
```

Comment:
This shouldn't override the defaults globally this will break people's experiments no? It should start at 0 and the task config should override it.